samples/Server/src/routes/uploadToAzureBlobStorage.ts

```diff
@@ -69,4 +74,78 @@ router.post('/', async function (req, res, next) {
   res.status(200).send(sasToken);
 });
 
+/**
+ * route: /uploadToAzureBlobStorage/file
+ *
+ * purpose: Upload a file to azure blob storage
+ * @param req.body: the form data containing the file
+ * @param req.params.filename: the filename of the file
+ * @param req.params.containerName: name of the container to upload to
+ *
+ * @returns SAS URL for accessing blob. This URL is valid for 1 hour.
+ *
+ */
+router.post('/file/:filename/:containername', async function (req, res, next) {
+  const SAS_EXPIRY = 1 * 60 * 60; // 1 hour
```

Comment:
I believe we have a constant.ts file? If yes, let's move it there

Reply:
we currently don't have a constant.ts file